Makefile

```diff
@@ -93,153 +90,163 @@ else
+	@docker-compose -f docker-compose.yml logs -f redis
 
 .PHONY: stack-logs-api
 stack-logs-api: ## prints out and follows the logs of the 'api' container running in knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml logs -f api
+	@docker-compose -f docker-compose.yml logs -f api
 
 .PHONY: stack-logs-api-no-follow
 stack-logs-api-no-follow: ## prints out the logs of the 'api' container running in knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml logs api
+	docker-compose -f docker-compose.yml logs api
 
 .PHONY: stack-logs-salsah1
 stack-logs-salsah1: ## prints out and follows the logs of the 'salsah1' container running in knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml logs -f salsah1
+	docker-compose -f docker-compose.yml logs -f salsah1
 
 .PHONY: stack-health
 stack-health:
 	curl -f 0.0.0.0:3333/health
 
 .PHONY: stack-status
 stack-status:
-	docker-compose -f docker/knora.docker-compose.yml ps
+	docker-compose -f docker-compose.yml ps
 
 .PHONY: stack-down
 stack-down: ## stops the knora-stack.
-	docker-compose -f docker/knora.docker-compose.yml down
+	docker-compose -f docker-compose.yml down
 
 .PHONY: stack-down-delete-volumes
 stack-down-delete-volumes: ## stops the knora-stack and delete any created volumes (deletes the database!).
-	docker-compose -f docker/knora.docker-compose.yml down --volumes
+	docker-compose -f docker-compose.yml down --volumes
 
 .PHONY: stack-config
 stack-config: env-file
-	docker-compose -f docker/knora.docker-compose.yml config
+	docker-compose -f docker-compose.yml config
 
 ## stack without api
 .PHONY: stack-without-api
 stack-without-api: stack-up ## starts the knora-stack without knora-api: graphdb, sipi, redis, salsah1.
-	docker-compose -f docker/knora.docker-compose.yml stop api
+	@docker-compose -f docker-compose.yml stop api
 
-## stack without api and sipi
 .PHONY: stack-without-api-and-sipi
 stack-without-api-and-sipi: stack-up ## starts the knora-stack without knora-api and sipi: graphdb, redis, salsah1.
-	docker-compose -f docker/knora.docker-compose.yml stop api
-	docker-compose -f docker/knora.docker-compose.yml stop sipi
+	@docker-compose -f docker-compose.yml stop api
+	@docker-compose -f docker-compose.yml stop sipi
 
-.PHONY: test-only
-test-only: build-all-images ## runs only the supplied tests, e.g., make test-only TARGET="*.CORSSupportE2ESpec".
-	@echo $@  # print target name
-	sbt "webapi/testOnly $(TARGET)"
+.PHONY: stack-without-api-and-salsah1
+stack-without-api-and-salsah1: stack-up ## starts the knora-stack without knora-api and salsah1: graphdb, redis, sipi.
+	@docker-compose -f docker-compose.yml stop api
+	@docker-compose -f docker-compose.yml stop salsah1
 
-.PHONY: test-unit
-test-unit: build-all-images ## runs the unit tests (equivalent to 'sbt webapi/testOnly -- -l org.knora.webapi.testing.tags.E2ETest').
-	@echo $@  # print target name
-	sbt 'webapi/testOnly -- -l org.knora.webapi.testing.tags.E2ETest'
+.PHONY: stack-db-only
```

Comment:
I have difficulty understanding why, in order to just start the `db` container, we need to start all then stop all except `db`?!

Reply:
I guess, at that time it was easier to implement like that. Sure, this can be simplified now.

Reply:
That would be really helpful, because starting all the containers is slow.

Reply:
(especially on my slow iMac)

Reply:
We should get you a fast notebook if it means less work for me ;-)

Reply:
And for Sepideh and Kalthoum, they have the same problem. I think your machine is much faster than everyone else's...

Reply:
I've optimized it a bit. It is difficult to do it in a generalized way. If you have a specific workflow that you call all the time, then we can try and add a target that does exactly what and how you need it.

Here are the new steps:

1. `make docker-build` # run at least once so that you have all Docker images
2. `make init-db-test` # stops webapi, deletes and runs only the DB container, creates the repository and loads the data
3. `make stack-up-fast` # starts the whole stack and only rebuilds the `knora-api` Docker container if necessary

After you made some changes to webapi:

1. `make init-db-test` # stops, webapi, deletes and runs only the DB container, creates the repository and loads the data
2. `make stack-up-fast` # starts the whole stack and only rebuilds the `knora-api` Docker container if necessary

Reply:
@subotic have you already implemented these new steps? I had made a YouTrack issue about this with suggestions
[DSP-463](https://dasch.myjetbrains.com/youtrack/issue/DSP-463)

Reply:
I did optimize it a bit. Your suggested way of doing it cannot be implemented at the moment. I've written a comment explaining it.